packs/pagerduty/actions/resolve_incident.yaml

```diff
@@ -0,0 +1,10 @@
+name: resolve_incident
+runner_type: run-python
+description: Resolve an incident whose key is provided
+enabled: true
+entry_point: resolve_incident.py
+parameters:
+  event_keys:
+    type: array
```

Comment:
Yeah, using the CLI you specify an array as a comma delimited string (inside the CLI string gets split into comma and converted into a list).

Will also add clarification on how to specify parameters which are a type of array to the CLI docs.